docs/testcases/TC06-010-fail.html

```diff
@@ -0,0 +1,195 @@
+									</ol>
+
+									<h4 id="test-instructions">Test Instructions</h4>
+									<ol>
+									<li>Check that the combination of the accessible name and accessible description is not empty. [SC 1.1.1]</li>
+									<li>Check that the non-empty combination of accessible name and accessible description provides an equivalent description. Numerous attributes contribute to the computation of the accessible name and accessible description. Refer to <ins><a href="https://www.w3.org/TR/html-aam-1.0/#img-element">HTML Accessibility API Mappings 1.0 for img</a></ins>. [SC 1.1.1]
+										<ol>
+										<li>Descriptions of the image that are provided by page content must be programmatically associated.</li>
+										<li>When an image is updated to convey a new meaning, check that its text alternative is updated at the same time. [SCs 1.1.1 and 4.1.2]</li>
+										</ol>
+									</li>
+									<li>Check that the ARIA role is NOT “presentation”.</li>
+									<li>Check that the ARIA role is NOT “none”.</li>
+									<li>Check that aria-hidden is NOT set to “true”.</li>
+									</ol>
+
+									<h4 id="test-results">Test Results</h4>
+									<ul>
+									<li>If any of the above checks fail, then Baseline Test 6.1-MeaningfulImage fails.</li>
+									</ul>
+								<h3 id="TCreq">Applicable WCAG/508 Requirement</h3>
+								<ul>
+									<li><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html">WCAG2 SC: 1.1.1. Non-Text</a> – All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for [specific] situations listed.</li>
+									<li><a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html">WCAG2 SC: 4.1.2 Name, Role, Value</a> – For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies.</li>
+								</ul>
+  					</div>
+  				</div>
+  				<div class="card shadow my-3">
+  					<div class="card-body">
+  						<h2 class="border-bottom" id="TCdet">Test Case Detail</h2>
+  							<h3 id="TCresult">Expected Baseline Result</h3>
+  								<p><strong>FAIL</strong></p>
+  								<p>The image in the code sample provides an ALT attribute with a value that DOES NOT provide an equivalent text alternative description of the image. </p>
+  							<h3 id="TCdata">Test Data (code/link)</h3>
+  								<h4 id="TCcode">Code</h4>
+  									<pre class="bg-light callout">
+  										<code>
+  &ltbody&gt
+	&ltimg src="../assets/images/corplogo.png" width="50" height="50"
+          class="" alt="pngfilelogo"&gt
+	&ltp&gtThis is a meaningful image&lt/p&gt
```

Comment:
Minor issue: The code <p> value does not match the actual <p> value in the embedded test file. The difference is minor but different.

Reply:
Fixed